simulation/validator/reward.py

```diff
@@ -44,10 +45,21 @@ def reward(
     - float: The reward value for the miner.
     """
 
-    miner_prediction_id, predictions = miner_data_handler.get_miner_prediction(
-        miner_uid, validator_request_id
+    miner_prediction_id, predictions, format_validation = (
+        miner_data_handler.get_miner_prediction(
+            miner_uid, validator_request_id
+        )
     )
 
+    if format_validation != response_validation.CORRECT:
+        return (
```

Comment:
nit: can we make it a one-liner like in other places of this function:
```
return -1, [], [], miner_prediction_id
```

or change other places like you make here, so it looks the same everywhere

Reply:
our formatter `black --line-length 79 --exclude '(env|venv|.eggs|.git)' --check .` wants multiple line because on the lines length